NeoML/include/NeoML/Dnn/Dnn.h

```diff
@@ -87,6 +87,7 @@ class CDnn;
 class CDnnLayerGraph;
 class CBaseLayer;
 class CCompositeLayer;
+class CDnnRefManager;
```

Comment:
`CDnnRefManager` -> `CReferenceDnnCounter` or `CReferenceDnnRegistor`or `CReferenceDnnManager`
 It is will be hard to remember the original goal of this